### PR TITLE
Fixed pagination bug

### PR DESCRIPTION
**Pull request by laurence-bird**

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -13,7 +13,7 @@ aws {
   }
   dynamo{
     table{
-      comms = "audit-log-UAT-CommsTable-11QXC9DG7R3QK"
+      comms = "audit-log-PRD-CommsTable-7ARMXPXJJBUH"
```
Comment (fleetwoods) [inline conf/application.conf]: Maybe best not PRD?

**Comment by fleetwoods** [comment]
Looks good other than the PRD table in local config

File: conf/application.conf (hunk 1 above)
Comment (laurence-bird) [inline conf/application.conf]: oops. my bad will change!

**Comment by ovo-comms-circleci** [comment]
This PR was successfully deployed to production by [CircleCI build 101](https://circleci.com/gh/ovotech/comms-audit-log/101) at 2017-02-28 17:31 GMT.

**Pull request merged**